### DISCUSSION TITLE
optimization in OMEdit

### DISCUSSION THREAD

**Discussion by mario35to** [OpenModelica/OpenModelica#11851] [Q&A]
hello all,

"optimization" command isn't supported in OMEdit, I only able to find a way to use optimization as below in https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/optimization.html;
annotation(experiment(StartTime = 0, StopTime = 1, Tolerance = 1e-07, Interval = 0.01),
__OpenModelica_simulationFlags(s="optimization", optimizerNP="1"),
__OpenModelica_commandLineOptions="+g=Optimica");

this way doesn't solve the optimization explicitly, only give information, as I understand.

Is there any alternative to do optimization in OMEdit?

**Comment by casella** [comment]
@ceraolo, you managed to do some simple optimization of battery-powered electrical vehicles with OMC, can you please comment on that?

**Comment by ceraolo** [comment]
I personality wrote the part of the manual you linked along with the guy that implemented the related part of SW,  @vruge.
It contains optimization examples that do actual dynamic optimisation, do not just give information. The last example is related to a hybrid-electric vehicle.
Please go ahead with the manual to see the examples.
More recently, I tried to use it to optimise EV charge with charge stations with storage and photovoltaic generation, but had troubles. So, not all my stories with DO in OM are successful.

**Reply by casella** [reply]
Please note that this feature is currently not maintained, so it is provided as is. It was promising, but the main developer switched to another job and we haven't had the resource to continue the development since then
